client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -93,24 +87,115 @@ function getHoldMessages( translate: LocalizeProps[ 'translate' ] ) {
 	};
 }
 
+/**
+ * This function defines how we should communicate each type of blocking hold the public-api returns.
+ * Blocking holds are "hard stops" - if we detect any, we know the Atomic Transfer won't be possible and so we
+ * should short-circuit any eligibility checks and just communicate the problem.
+ *
+ * @param {Function} translate Translate fn
+ * @returns {object} Dictionary of blocking holds and their corresponding messages
+ */
+function getBlockingMessages( translate: LocalizeProps[ 'translate' ] ) {
+	return {
+		BLOCKED_ATOMIC_TRANSFER: {
+			message: hasTranslation(
+				'This site is not currently eligible to install themes and plugins, or activate hosting access. Please contact our support team for help.'
+			)
+				? translate(
+						'This site is not currently eligible to install themes and plugins, or activate hosting access. Please contact our support team for help.'
+				  )
+				: translate(
+						'This site is not currently eligible to install themes and plugins. Please contact our support team for help.'
+				  ),
+			status: 'is-error',
+			contactUrl: localizeUrl( 'https://wordpress.com/help/contact' ),
+		},
+		TRANSFER_ALREADY_EXISTS: {
+			message: translate(
+				'Installation in progress. Just a minute! Please wait until the installation is finished, then try again.'
+			),
+			status: null,
+			contactUrl: null,
+		},
+		NO_JETPACK_SITES: {
+			message: translate( 'Try using a different site.' ),
+			status: 'is-error',
+			contactUrl: null,
+		},
+		NO_VIP_SITES: {
+			message: translate( 'Try using a different site.' ),
+			status: 'is-error',
+			contactUrl: null,
+		},
+		SITE_GRAYLISTED: {
+			message: hasTranslation(
+				"There's an ongoing site dispute. Contact us to review your site's standing and resolve the dispute."
+			)
+				? hasTranslation(
+						"There's an ongoing site dispute. Contact us to review your site's standing and resolve the dispute."
+				  )
+				: translate( "Contact us to review your site's standing and resolve the dispute." ),
+			status: 'is-error',
+			contactUrl: localizeUrl( 'https://en.support.wordpress.com/suspended-blogs/' ),
+		},
+		NO_SSL_CERTIFICATE: {
+			message: hasTranslation(
+				'Certificate installation in progress. Hold tight! We are setting up a digital certificate to allow secure browsing on your site using "HTTPS".'
+			)
+				? hasTranslation(
+						'Certificate installation in progress. Hold tight! We are setting up a digital certificate to allow secure browsing on your site using "HTTPS".'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Hold tight! We are setting up a digital certificate to allow secure browsing on your site, using "HTTPS". Please try again in a few minutes.' )` *ES Score: 9*